### DISCUSSION TITLE
Image preview is not working for few documents on search result page using modern pnp search filter

### DISCUSSION THREAD

**Discussion by DharmendraYadav90** [microsoft-search/pnp-modern-search#3592] [Q&A]
Describe the bug
When using the PnP Search Results Webpart and selecting SharePoint Search as the data source, we get number of documents, sites, lists on the page, however the preview does not work for few of the result.

To Reproduce

Go to a SharePoint Online page.
Add a PnP Search Results Webpart
Edit the webpart and select "SharePoint Search" as a data source.
The result will immediately show on the page, some with image and few without any Image preview.
Also, go to 2nd page in the property pane, and select Debug from the template layout.
AutoPreviewImageURL shows null value. 

Expected behavior
Search results should be shown with the Image preview.

**Comment by wobba** [comment]
This is not a web part bug per day. And you share no details of the actual error. You can open dev tools in your browser and see what happens on the failed thumbnail requests. This may help figure out why it happens. Do you get thumbnails for the same files in Microsoft Search experiences?
